functions.qmd

```diff
@@ -717,14 +729,15 @@ hex_plot <- function(df, x, y, z, bins = 20, fun = "mean") {
       fun = fun,
     )
 }
+
 diamonds |> hex_plot(carat, price, depth)
 ```
 
 ### Combining with dplyr
 
 Some of the most useful helpers combine a dash of dplyr with ggplot2.
 For example, if you might want to do a vertical bar chart where you automatically sort the bars in frequency order using `fct_infreq()`.
-Since the bar chart is vertical, we also need to reverse the usual order to get the highest values at the top:
+Since the bar chart is vertical, we also need to reverse the usual order to get the highest values at the top (also note the `:=` operator, which allows you to inject names with glue syntax on the left-hand side of ⁠`:=`;⁠ type: ?\`:=\` for more details):
 
```

Comment:
This is just a suggestion but the ":=" is new (I haven't seen it myself either before). So perhaps pointing it out may be a good idea? My edit here is just a suggestion. Doesn't have to be that way.

Reply:
I'll flesh this out a bit more after merging.